gnuplot/examples/box_and_whisker.rs

```diff
@@ -10,14 +10,22 @@ fn example(c: Common)
 
 	fg.axes2d()
 		.set_title("Box and whisker", &[])
-		.box_and_whisker_set_width(
+		.box_and_whisker(
```

Comment:
This looks like a bad merge. With this fix, this example's output should be unchanged.

Reply:
Yes, bad merge. Not sure why I didn't spot it though!